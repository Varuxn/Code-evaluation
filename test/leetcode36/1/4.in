10
-5 -1 3 1 0 -7 3 4 -7 -2 