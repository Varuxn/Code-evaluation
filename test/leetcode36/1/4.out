-7 -7 
-7 -7 -2 
-7 -2 
-7 3 
-7 3 4 
-7 4 
-5 -2 
-5 -1 
-5 -1 0 
-5 -1 0 3 
-5 -1 0 3 4 
-5 -1 0 4 
-5 -1 1 
-5 -1 1 3 
-5 -1 1 3 4 
-5 -1 1 4 
-5 -1 3 
-5 -1 3 3 
-5 -1 3 3 4 
-5 -1 3 4 
-5 -1 4 
-5 0 
-5 0 3 
-5 0 3 4 
-5 0 4 
-5 1 
-5 1 3 
-5 1 3 4 
-5 1 4 
-5 3 
-5 3 3 
-5 3 3 4 
-5 3 4 
-5 4 
-1 0 
-1 0 3 
-1 0 3 4 
-1 0 4 
-1 1 
-1 1 3 
-1 1 3 4 
-1 1 4 
-1 3 
-1 3 3 
-1 3 3 4 
-1 3 4 
-1 4 
0 3 
0 3 4 
0 4 
1 3 
1 3 4 
1 4 
3 3 
3 3 4 
3 4 
